src/bucket/BucketSnapshotManager.cpp

```diff
@@ -231,4 +225,11 @@ BucketSnapshotManager::endPointLoadTimer(LedgerEntryType t,
         iter->second.Update(duration);
     }
 }
+
+uint32_t
+BucketSnapshotManager::getCurrentLedgerSeq() const
```

Comment:
+1 to Dima. I think this function is general is a little unsafe as it leads to non-atomic updates.